docs/types/reference-types.rst

```diff
@@ -120,7 +120,7 @@ but it is packed tightly in calldata and memory. ``string`` is equal to ``bytes`
 length or index access.
 
 While Solidity does not have string manipulation functions, you can use
-this implicit conversion for equivalent functionality. For example to compare
+this array equivalence for similar functionality. For example to compare
```

Comment:
I still don't understand the connection between this paragraph and the preceding one.

Reply:
Because the fact that `bytes` and `strings` are special forms of arrays is what leads to this equivalent functionality. Can you maybe suggest a better location if you don't think this is the right one?